packages/orbit-components/src/Card/index.tsx

```diff
@@ -48,46 +27,34 @@ const Card = ({
   header,
   spaceAfter,
   dataA11ySection,
-}: Props) => {
-  const [expandedSections, setExpandedSections] = React.useState<number[]>([]);
-  const theme = useTheme();
-
-  // handles array of expanded sections
-  const addSection = React.useCallback((index: number) => {
-    setExpandedSections(prev => (prev.indexOf(index) === -1 ? [...prev, index] : prev));
-  }, []);
-
-  const removeSection = React.useCallback((index: number) => {
-    setExpandedSections(prev =>
-      prev.indexOf(index) !== -1 ? prev.filter(val => val !== index) : prev,
-    );
-  }, []);
-
-  // Currently disable that code, becuase of IE 11, where it does not work
-  // It will be fixed later, when we'll find solution
-  const renderSection = item => {
-    if (React.isValidElement(item)) {
-      // if (item.props.children && item.type.name !== "CardSection") {
-      //   return React.createElement(CardSection, {
-      //     ...item.props.children.props,
-      //     key: index,
-      //   });
-
-      return React.cloneElement(item);
+}: Props) {
+  const marginStyles = (() => {
+    if (margin == null) {
```

Comment:
let's not add curly braces if this can be one line return :) 
Not a big deal and more like a matter of personal preference, we do not have some eslint rules to enforce that and do not think we should, but this is how we get used to write in this project :)

Reply:
I would enforce that with multi-option, meaning, that it will add curlies in case it's multiline. We have used it here for years and no bugs were related to that.

Reply:
let's solve it separately, though imo we should do `all` for maximum consistency and readability